client/blocks/nav-unification-modal/style.scss

```diff
@@ -0,0 +1,237 @@
+// @TODO: remove the ignore rule and replace font sizes accordingly
+/* stylelint-disable scales/font-size */
+
+@import '~@automattic/typography/styles/fonts';
+@import '~@automattic/onboarding/styles/mixins';
+@import '~@wordpress/base-styles/breakpoints';
+
+$wpcom-modal-breakpoint: 600px;
```

Comment:
Is there any reasoning for `wpcom` in the naming? And if so, would it provide any value documenting it inline?